src/kademlia/item.cpp

```diff
@@ -69,7 +68,6 @@ namespace
 			left = out.size() - (ptr - out.data());
 			std::memcpy(ptr, salt.data(), (std::min)(salt.size(), left));
 			ptr += (std::min)(salt.size(), left);
-			left = out.size() - (ptr - out.data());
```

Comment:
``left`` is used after this, I don't think this is safe to remove. note that ``ptr`` will have a new value in this expression than the one two lines up